docs/troubleshooting.md

```diff
@@ -19,6 +19,17 @@ If you attempted to deploy Wildebeest in your account and something failed, or y
 
 You can now start a clean install.
 
+### Finding error causes
+
+If you get an error page or 500 status error for the main HTML document you can find the source of the error using the wrangler CLI:
+
+- go to your Cloudflare account dashboard > Pages and click on `wildebeest-YOURNAME`
+- click on the Production deployment for your project (View details)
+- copy the second UUID in the URL, which is your deployment ID
+- get wrangler CLI if you don't already have it (`yarn global add wrangler`)
+- run `wrangler pages deployment tail DEPLOYMENT_ID_HERE`
```

Comment:
the deployment id is optional and people can just do `wrangler pages deployment tail`, then select their project via the interactive CLI and it will tail their latest deployment, maybe we should suggest that as it sounds simpler got newcomers?

Reply:
I got an error if I didn't specify the deployment ID. Did that change recently?

Reply:
hmm can't reproduce now, will update 👍🏻